### PR TITLE
Add Readme and delete unused files.

### PR DESCRIPTION
**Pull request by Jaesang**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,72 @@
-# tacoctl
+# tacoctl 
 CLI tool for TACOPLAY
+
+------
+
+## What is the tacoctl
+
+The tacoctl is next deploy management tool of [tacoplay](github.com/openinfradev/tacoctl). It support a Cluster
```
Comment (bluejayA) [inline README.md]: The tacoctl project is a CLI tool for declarative management of kubernetes and underlying infrastructure, mainly leveraging the Kubernetes Cluster API. It builds and manage a Kubernetes Cluster either on Baremetal or OpenStack VMs. 

This project is the ongoing effort to produce next generation of existing [tacoplay](github.com/openinfradev/tacoplay). Currently, the tacoctl is at an very early development stage.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,72 @@
-# tacoctl
+# tacoctl 
 CLI tool for TACOPLAY
```
Comment (bluejayA) [inline README.md]: CLI tool for Declarative Lifecycle Management of Kubernetes Cluster

**Review by bluejayA — COMMENTED** [review]
Introduction 부분은 다시 써 보았습니다.

File: README.md [hunk 3]
```diff
@@ -1,2 +1,72 @@
...
+
+### Add Worker Node
+
+```sh
+tacoctl create node
+```
+
+## Features
+
+- Manage a lifecycle a Kubernetes cluster on OpenStack.
```
Comment (bluejayA) [inline README.md]: the lifecycle of a Kubernetes cluster

File: README.md [hunk 4]
```diff
@@ -1,2 +1,72 @@
...
+
+## Roadmap
+
+- Manage a lifecycle a Kubernetes cluster on Barematal.
```
Comment (bluejayA) [inline README.md]: the lifecycle of a Kubernetes cluster on Baremetal

**Comment by Jaesang** [comment]
수정사항 반영해서 다시 올렸습니다.

**Pull request merged**